Source/init.cpp

```diff
@@ -164,6 +165,10 @@ void init_create_window(int nCmdShow)
 		nHeight = SCREEN_HEIGHT;
 	else
 		nHeight = GetSystemMetrics(SM_CYSCREEN);
+#else
+	nWidth = 640;
+	nHeight = 480;

```

Comment:
Might be better to either disable upscaling or modify GetSystemMetrics instead.

Reply:
fixed in https://github.com/diasurgical/devilutionX/pull/201/commits/b988cf99db11a935a27790613db0f06d836e6598

in PR #201